diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/DiktatRuleSetProvider.kt

```diff
@@ -46,34 +46,36 @@ class DiktatRuleSetProvider(private val diktatConfigFile: String = "diktat-analy
             ?.onEach(::validate)
             ?: emptyList()
         val rules = listOf(
+                // comments & documentation
                 ::CommentsRule,
                 ::KdocComments,
                 ::KdocMethods,
                 ::KdocFormatting,
+                ::CommentsFormatting,
+                // naming
                 ::FileNaming,
                 ::PackageNaming,
-                ::StringTemplateFormatRule,
-                ::FileSize,
-                ::DataClassesRule,
                 ::IdentifierNaming,
-                ::LocalVariablesRule,
+                // code structure
```

Comment:
may be you will add a comment so we won't change the order of the code in the future?

Reply:
Yes, good point